Dockerfile

```
@@ -1,5 +1,35 @@
+FROM node:14.14.0 as node-build
+
+ADD . /hydroshare
+
+WORKDIR /hydroshare/hs_discover
+
+RUN rm -rf static templates && \
+    mkdir static templates && \
+    mkdir templates/hs_discover && \
+    mkdir static/js && \
+    mkdir static/css && \
+    npm install && \
+    npm run build && \
+    mkdir -p static/js && \
+    mkdir -p static/css && \
```

these `mkdir` seem redundant
```suggestion
    mkdir -p static/js && \
    mkdir -p static/css && \
    npm install && \
    npm run build && \
```